.github/ISSUE_TEMPLATE/new_action.yml

```diff
@@ -12,3 +12,14 @@ body:
         Ex. It would be great to have an action that would ...
     validations:
       required: false
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Items that need to be completed along with the creation of the new App or Action
```

Comment:
```suggestion
      description: Items that need to be completed along with the creation of the new App or Action (to be filled out later when the action is being developed)
```

Reply:
Does this add clarity that folks shouldn't think that they need to do these things as perquisites to opening the issue or is there another way to add that clarity?

Reply:
or maybe 
```suggestion
      description: Items that should be completed during the development of the new App or Action
```